web/scripts/generateSitemap.js

```diff
@@ -25,7 +30,12 @@ function generateSitemap() {
   }
 
   const zoneUrls = Object.keys(zonesConfig.zones)
-    .map((zone) => `<url><loc>https://app.electricitymaps.com/zone/${zone}</loc></url>`)
+    .flatMap((zone) =>
```

Comment:
nit: Not sure if it matter much, but the `UrlTimeAverages` also get appended to /map (example: https://bbb1b2b8.electricitymaps-staging.pages.dev/map/12mo)

Reply:
I'll make sure to add them 👍🏼